web/app/view/dialog/ReportConfigController.js

```diff
@@ -26,23 +26,25 @@ Ext.define('Traccar.view.dialog.ReportConfigController', {
     ],
 
     onSaveClick: function (button) {
-        var eventType;
-        this.getView().callingPanel.deviceId = this.lookupReference('deviceField').getValue();
-        this.getView().callingPanel.groupId = this.lookupReference('groupField').getValue();
+        var eventType,callingPanel;
```

Comment:
I have both packages you mention installed and I can see them working as now if i force an error it shows notes from both.